stdlib/2and3/argparse.pyi

```diff
@@ -283,6 +284,21 @@ class Action(_AttributeHolder):
     if sys.version_info >= (3, 9):
         def format_usage(self) -> str: ...
 
+if sys.version_info >= (3, 9):
+    class BooleanOptionalAction(Action):
+        def __init__(
+            self,
+            option_strings: Sequence[Text],
+            dest: Text,
+            const: Any = ...,
+            default: Any = ...,
+            type: Optional[Union[Callable[[Text], _T], Callable[[str], _T], FileType]] = ...,
```

Comment:
Should this class be generic over _T?

Reply:
I don't think so? Not sure what that would allows us to accomplish